message/message1_1prime/schema.ipldsch

```diff
@@ -35,3 +36,8 @@ type TransferMessage struct {
 	Request  nullable TransferRequest
 	Response nullable TransferResponse
 }
+
+type WrappedTransferMessage struct {
+	TransportID TransportID (rename "ID")
```

Comment:
So how strict can we be here? I assume that because we have an explicit list of transports we support that we get to be very strict with the values that can appear here. Because if we can, then maybe this should be a keyed union:

```
type WrappedTransferMessage union {
   Message "1.2.0"
} representation keyed
```

We get more compact representation and very strict matching on allowable version(s), and we can adjust the `Message` format with version changes.